### PR TITLE
marcus upadte

### PR DESCRIPTION
**Pull request by ethanlchristensen**

### CODE REVIEW

File: BOT/slash_master.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def __init__(self, tree, guild, path, debug):
         if platform == "linux" or platform == "linux2" or platform == "darwin":
             self._path = os.path.dirname(__file__) + "/" + path
         else:
-            self._path = os.path.dirname(__file__) + path
+            self._path = os.path.dirname(__file__) + "/" + path
```
Comment (nulzo) [inline BOT/slash_master.py]: no

Comment (ethanlchristensen) [reply BOT/slash_master.py]: ?

Comment (nulzo) [reply BOT/slash_master.py]: self._path = os.path.dirname(\_\_file\_\_) + "\\\" + path

Comment (ethanlchristensen) [reply BOT/slash_master.py]: image

Comment (ethanlchristensen) [reply BOT/slash_master.py]: image

**Review by nulzo — APPROVED** [review]

**Pull request merged**